internal/models/account_id_test.go

```diff
@@ -0,0 +1,167 @@
+package models_test
+
+import (
+	"testing"
+
+	"github.com/formancehq/payments/internal/models"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAccountID(t *testing.T) {
+	t.Parallel()
+
+	t.Run("String", func(t *testing.T) {
+		t.Parallel()
+		
+		// Given
```

Comment:
In this whole PR, you should never have test with only "Given"
I expect to see tests with 1 Given, 1 When and 1 Then, annotated as comment

Given section shoudl contain test section
When section -- usually 1 liner -- is the call being tested
Then section -- the various assertions.